vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -4,53 +4,69 @@
 #pragma once
 
 #include "ChakraObjectRef.h"
-#include "ChakraRuntimeArgs.h"
 
 #include "jsi/jsi.h"
 
-#ifdef CHAKRACORE
-#include "ChakraCore.h"
-#include "ChakraCoreDebugger.h"
-#else
-#ifndef USE_EDGEMODE_JSRT
-#define USE_EDGEMODE_JSRT
-#endif
-#include "jsrt.h"
-#endif
-
 #include <memory>
-#include <mutex>
-#include <sstream>
-
-#if !defined(CHAKRACORE)
-class DebugProtocolHandler {};
-class DebugService {};
-#endif
+#include <string>
+#include <type_traits>
+#include <utility>
+#include <vector>
 
 namespace Microsoft::JSI {
 
+struct ChakraRuntimeArgs;
+
+class ChakraPreparedJavaScript : public facebook::jsi::PreparedJavaScript {
```

Comment:
>ChakraPreparedJavaScript [](start = 6, length = 24)

nit: This header seems to be getting big. Would this be better in its own file?

Reply:
I agree. Will fix this in a future iteration.

---
In reply to: [345451951](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345451951) [](ancestors = 345451951)